packages/block-editor/src/components/off-canvas-editor/leaf-more-menu.js

```diff
@@ -19,11 +19,69 @@ const POPOVER_PROPS = {
 	variant: 'toolbar',
 };
 
+const BLOCKS_THAT_CAN_BE_CONVERTED_TO_SUBMENU = [
+	'core/navigation-link',
+	'core/navigation-submenu',
+];
+
+function AddSubmenuItem( { block, clientId, onClose } ) {
```

Comment:
Isn't `clientId` a property of `block`? Could we refactor to remove arguments?

Reply:
done